source/grammar/openpulse1.g4

```diff
@@ -0,0 +1,74 @@
+/**** ANTLRv4  grammar for OpenPulse1.0.
+
+The following grammar specifies valid OpenPulse statements within the .*? of
+OpenQASM3 below.
+
+calibrationDefinition
+    : 'defcal' calibrationGrammar? Identifier
+    ( LPAREN calibrationArgumentList? RPAREN )? identifierList
+    returnSignature? LBRACE .*? RBRACE  // for now, match anything inside body
+    ;
+
+specifically, `.*? -> body` below.
+
+****/
+
+grammar openpulse1;
+
+body
+    : (statement)+
+    ;
+
+statement
+    : classicalAssignment
+    | classicalDeclaration
+    | constantDeclaration
+    | aliasStatement
+    | branchingStatement
+    | instruction
+    | pragma
+    ;
+
```

Comment:
Should we have a pulse library or shape library? The standard envelopes like `gaussian`, `gaussian_square`, etc are referenced in https://github.com/Qiskit/openqasm/pull/109/files.